e2e/test/MessageSendE2ETests.cs

```diff
@@ -30,7 +30,8 @@ public MessageSendE2ETests()
         [TestMethod]
         public async Task Message_DeviceSendSingleMessage_Amqp()
         {
-            await SendSingleMessage(TestDeviceType.Sasl, Client.TransportType.Amqp_Tcp_Only).ConfigureAwait(false);
+            //await SendSingleMessage(TestDeviceType.Sasl, Client.TransportType.Amqp_Tcp_Only).ConfigureAwait(false);
+            await SendMultipleMessages(TestDeviceType.Sasl, Client.TransportType.Amqp_Tcp_Only).ConfigureAwait(false);
```

Comment:
Why is SendSingleMessage commented? This looks to be a _"testing"_ code that needs to be reverted.

Reply:
reverted, checking in now

Reply:
What is the scenario that we are trying the verify here? If _send_ works for a single message, it would work for >1 messages as well.

Reply:
Adding @prmathur-microsoft , she has more context on why we need multiple message test.

Reply:
On a second thought, we test it in new e2e combined tests for parallel tests on send and receive. Hence removing these additional tests.